service/templates/projects.html

```diff
@@ -399,7 +399,7 @@ <h2> Settings for [[displayName]]</h2>
           <div class="form-group">
             <div class="col-header col-lg-6 form-control-static"> 
               <h3>Commit Queue</h3> 
-              <div class="muted small">The commit queue merges changes onto the project's branch after they've passed a set of tests.</div>
+              <div class="muted small">The commit queue merges changes onto the project's branch after they've passed a set of tests. Can only be enabled if patch definitions are set.</div>
```

Comment:
maybe this addition should be on the enable checkbox?

Reply:
I'd like to leave it here; conditions shouldn't be set with the label of a checkbox, and this is still right above the checkbox.